.github/workflows/inspektor-gadget.yml

```diff
@@ -225,8 +228,25 @@ jobs:
             IMAGE_TAG="latest"
         fi
         echo IMAGE_TAG=$IMAGE_TAG >> $GITHUB_ENV
+    - name: Cross build gadget container for amd64 and arm64.
+      uses: docker/build-push-action@v2
+      # We cross build only on release because it takes a lot of time.
+      if: github.ref == 'refs/heads/francis/arm'
```

Comment:
It is on purpose and marked as a TODO in the cover letter:
>  Change ref/heads to main.

I will change this right before merging.
For now, it will remain like this, so I am sure it is cross built when I push there.

Reply:
It's not about changing `'refs/heads/francis/arm'` to `'refs/heads/main'`, it's about adding another condition to run this logic when a tag is pushed, hence building the arm image a release too.